storage/remote/queue_manager.go

```diff
@@ -97,7 +124,6 @@ func NewStorageQueueManager(tsdb StorageClient, cfg *StorageQueueManagerConfig)
 				Namespace:   namespace,
 				Subsystem:   subsystem,
 				Name:        "sent_samples_total",
```

Comment:
You keep the locally defined metrics here but just delete their help string... probably not what you intended.

Honestly I'd just leave the metrics structure like it is. The `ConstLabels` that we had here would include the right differentiated `type` already when `tsdb.Name()` includes the hostname or whatever sufficiently differentiated suffix we chose.